vector/src/main/res/xml/vector_settings_preferences.xml

```diff
@@ -198,6 +198,16 @@
 
     </im.vector.app.core.preference.VectorPreferenceCategory>
 
+    <im.vector.app.core.preference.VectorPreferenceCategory android:title="@string/settings_presence">
+
+        <im.vector.app.core.preference.VectorSwitchPreference
+            android:title="@string/settings_show_presence"
+            android:summary="@string/settings_show_presence_summary"
+            android:key="SETTINGS_SHOW_PRESENCE"
+            android:defaultValue="true" />
```

Comment:
The default value here should be the default value you defined in vector-config I guess